docs/guide/programming-tasks.md

```diff
@@ -322,10 +328,11 @@ Example:
 load("cirrus", "changes_include_only")
 
 def main(ctx):
+    # skip if only documentation changed
     if changes_include_only("doc/*"):
         return []
 
-    return base_tasks()
```

Comment:
Maybe we shouldn't remove this to provide a better example of what is going to happen when no changes were detected?

Reply:
I think it's fine. The `# ...` below kind of tips that you can write some default stuff.